tests/fixtures.py

```diff
@@ -27,9 +27,8 @@
 import mock
 import multiprocessing
 import time
-import urllib
 import socketio
-import mslib.mswms.mswms
+import mslib.mswms.wms
```

Comment:
the server setup is based on the wms

Reply:
I don't mind, lets try how mswms now behaves

Reply:
Both are the same object, so it shouldn't matter.